### PR TITLE
Update 07-vnc-server.md

### PR DESCRIPTION
**Pull request by Amiga500**

### CODE REVIEW

**Comment by XK9274** [comment]
Hey @Amiga500 - thanks for all of your contributions and additions to the docs, it's hugely appreciated by all of us here. 

Would it be possible for you to merge all of your docs PR's into one single PR please? I know it's easier on web to do a PR for each but it causes us overhead (and spams our trackers)

If you need a hand with anything give me a shout, or if you're unable to, that's fine - we'll do it this time.

**Comment by Amiga500** [comment]
PR created and made some clean

File: website/docs/07-apps/03-network-features/07-vnc-server.md [hunk 1]
```diff
@@ -3,27 +3,27 @@ slug: /network/vnc
...
 ## Enabling VNC access
 
 You'll find it in Tweaks app: [`Network` -› `VNC: Screen share`](/docs/apps/tweaks#vnc-screen-share).
 
-You can either use the master toggle to turn the VNC server on with your Right/Left arrows, or press A to enter the submenu and you'll get 2 options:
+You can either use the master toggle to turn the VNC Server on with your right/left arrows, or press A to enter the submenu and you'll get 2 options:
 
 1. **Enable** - Activate VNC Server
-2. **Framerate** - Set the framerate of the VNC Server between 1 and 20. The higher the framerate, the more CPU will be use.
+2. **Framerate** - Set the framerate of the VNC Server between 1 and 20. The higher the framerate, the more CPU will be use
```
Comment (XK9274) [inline website/docs/07-apps/03-network-features/07-vnc-server.md:26]: You ok to add the "d" on the end of "use" (used) then this one's good to go - Thanks!

**Review by XK9274 — CHANGES_REQUESTED** [review]

File: website/docs/07-apps/03-network-features/07-vnc-server.md (hunk 1 above)
Comment (Amiga500) [reply website/docs/07-apps/03-network-features/07-vnc-server.md]: Added on main PR. I close this one. Thanks!

**Pull request closed without merging**